scripts/asm-installer/tests/setup_longterm_cluster

```diff
@@ -26,12 +26,14 @@ setup_cluster() {
     --print-config \
     -D "${OUTPUT_DIR}" > /dev/null
 
+  cleanup_old_instance_templates
+  cleanup_old_instances
+  cleanup_old_images
   if [[ -n "${RESULT}" ]]; then
     echo "Long term test cluster ${cluster} exists already."
     configure_kubectl "${cluster}" "${LT_PROJECT_ID}" "${LT_CLUSTER_LOCATION}"
     kubectl delete validatingwebhookconfiguration istiod-istio-system || true
     cleanup_old_test_namespaces "${OUTPUT_DIR}"
-    cleanup_old_instance_templates
     cleanup_old_workload_service_accounts
```

Comment:
Let's move this function call out of if as well.

Reply:
updated